projects/storefrontlib/src/translations/en/add-to-cart.en.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  labels: {
```

Comment:
It's a bagatelle, but looks better: can we make singular the grouping keys
- label
- action
- validation
- placeholder